### PR TITLE
fix: replace "Github" with canonical "GitHub"

### PR DESCRIPTION
**Pull request by shuuji3**
# Pull request summary

Hi, this PR replaces "Github" with the canonical "GitHub" (with capitalized "H") in two places.

(Skipped creating an issue as this is a small fix.)

Screenshot 2024-01-26 at 23 43 09
Screenshot 2024-01-27 at 0 18 16

## Reminder - please do the following before assigning reviewer

- [ ] update readme
- [ ] For frontend changes, ensure design review

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introdcued

### CODE REVIEW

File: _data/social_media.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-- platform: "Github"
-  link: "https://github.com/18F" 
+- platform: "GitHub"
```
Comment (shuuji3) [inline _data/social_media.yml:1]: This `platform` value is used in `social-media.html`: https://github.com/shuuji3/18f.gsa.gov/blob/abac7fa0ca2faac8caf5fffa8c2e2ab833ea46c8/_includes/social-media.html#L12-L12
and rendered as `18F on GitHub`.

**Comment by shuuji3** [comment]
[CircleCI is failing here](https://app.circleci.com/pipelines/github/18F/18f.gsa.gov/2283/workflows/932f5ed9-74ba-4740-b776-ed1c6a221e2f/jobs/6491?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary&invite=true#step-104-228_60).

This is because the CI uses Ruby v3.2 image (https://github.com/18F/18f.gsa.gov/blob/main/.circleci/config.yml#L6) and it installs the latest Ruby v3.2.3, released on 2023-01-18 [Release note](https://www.ruby-lang.org/en/news/2024/01/18/ruby-3-2-3-released/)) but https://github.com/18F/18f.gsa.gov/blob/main/Gemfile#L3 specifies v3.2.2 explicitly.

**Comment by juliaklindpaintner** [comment]
@kbighorse Have you been able to make any progress on this one?

**Review by adunkman — APPROVED** [review]
Thanks for the fix, @shuuji3!

**Comment by adunkman** [comment]
(Oops, sorry to step on your footprint @kbighorse — please continue forward! I had this tab open for a bit and didn’t see the assignment above before reviewing).

**Pull request merged**